src/routes/categories/containers/CategoriesContainer.js

```diff
@@ -17,15 +17,15 @@ const mapStateToProps = (state: StateType, ownProps) => {
   const targetCity: string = ownProps.navigation.getParam('city')
   const targetPath: string = ownProps.navigation.getParam('path') || `/${targetCity}/${language}`
 
-  const cities = state.cities.json
+  const cities = state.cities
 
-  if (!cities) {
+  if (!cities.json || cities.error) {
```

Comment:
A check for `!cities` usually doesn't happed, but can happen when the state gets rehydrated in a wrong way.

Reply:
Change to `  if (!cities || !cities.json || cities.error) {`